recipes-bsp/device-tree/files/uz7ev-evcc-hdmi-v-n/hdmi.dtsi

```diff
@@ -0,0 +1,106 @@
+#include <dt-bindings/media/xilinx-vip.h>
+
```

Comment:
Looks like this file is similar to 'uz7ev-evcc-hdmi/hdmi.dtsi'
So it is not needed, the build system will use uz7ev-evcc-hdmi/hdmi.dtsi